### PR TITLE
Feature/56163 2

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: src/molecules/modal/Modal.html [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+    :role="modalRole"
   >
     <div
       @click.self="handleBackgroundClick"
       class="a-modal__mask"
     ></div>
     <transition :name="modalContentTransition">
       <div
-        class="a-modal__content"
+        :class="['a-modal__content', modalContentClass]"
```
Comment (szafran89) [inline src/molecules/modal/Modal.html]: do we need `modalContentClass` prop?

File: src/molecules/modal/Modal.selectors.json [hunk 2]
```diff
@@ -2,5 +2,21 @@
   {
     "name": ".a-modal--blank",
     "description": "Remove default content paddings"
+  },
+  {
+    "name": ".a-modal--compact",
```
Comment (szafran89) [inline src/molecules/modal/Modal.selectors.json]: do we need this modifier?

Comment (robwozniak) [reply src/molecules/modal/Modal.selectors.json]: It was taken from the old code, I can't find in designs what was the reason of this modifier so I'll remove it - if needed in future then we can easily add appropriate styles on higher component level

File: src/molecules/modal/Modal.html (hunk 1 above)
Comment (robwozniak) [reply src/molecules/modal/Modal.html]: Imo it's beneficial but for future usage, now it's not really needed - not used anywhere

**Pull request merged**